### PR TITLE
feat(authentication):  Add logout info to authentication doc

### PR DESCRIPTION
**Pull request by magsharon**
Add logout info to authentication doc.

Ticket Link: https://www.notion.so/immersve/Logout-docs-1000b3bdae7e4d12bb0f1c4798a69394

### CODE REVIEW

File: imsv-docs-astro/src/content/docs/guides/authentication.mdoc [hunk 1]
```diff
@@ -154,3 +154,20 @@ be exchanged when the new access token expires.
...
+
+### Logout Flow
+
+Initiate Logout Request: The client application sends a logout request to the [Logout API](/api-reference/logout), including the current access token for validation.
+
+Token Invalidated: Immersve invalidates the provided access token, ending the current session and ensuring that the token cannot be used for further requests.
+
+Client Cleanup: The client application clears any stored session information, such as the access token and refresh token, effectively logging the user out.
+
+<!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764582094189185&cot=14 -->
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/authentication.mdoc:172]: I think the sequence diagram is not adding value. It's just one API call.

Comment (magsharon) [reply imsv-docs-astro/src/content/docs/guides/authentication.mdoc:172]: I see. I will remove it.

Comment (magsharon) [reply imsv-docs-astro/src/content/docs/guides/authentication.mdoc]: sequence diagram is removed

File: imsv-docs-astro/src/content/docs/guides/authentication.mdoc [hunk 2]
```diff
@@ -154,3 +154,18 @@ be exchanged when the new access token expires.
 
 <!-- https://miro.com/app/board/uXjVNxzJMjE=/?moveToWidget=3458764581446931419&cot=14 -->
 ![Refresh Token Sequence Diagram](@assets/diagrams/refresh-token-sequence.svg)
+
+
+## Logout
+
+Logging out of the application involves invalidating the current access token to ensure it can no longer be used for accessing protected resources within the Immersve API.
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/authentication.mdoc:161]:
```suggestion
Logging out of the application involves invalidating the current session to ensure it can no longer be used for accessing protected resources within the Immersve API. All access tokens connected with the session will become unusable.
```

Comment (magsharon) [reply imsv-docs-astro/src/content/docs/guides/authentication.mdoc:161]: done ✅

File: imsv-docs-astro/src/content/docs/guides/authentication.mdoc [hunk 3]
```diff
@@ -154,3 +154,18 @@ be exchanged when the new access token expires.
...
+
+
+### Logout Flow
+
+Initiate Logout Request: The client application sends a logout request to the [Logout API](/api-reference/logout), including the current access token for validation.
+
+Token Invalidated: Immersve invalidates the provided access token, ending the current session and ensuring that the token cannot be used for further requests.
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/authentication.mdoc:168]:
```suggestion
Session Invalidated: Immersve invalidates the session and ensures that all access tokens related to the session are rejected by future requests.
```

File: imsv-docs-astro/src/content/docs/guides/authentication.mdoc [hunk 4]
```diff
@@ -154,3 +154,18 @@ be exchanged when the new access token expires.
...
+
+Client Cleanup: The client application clears any stored session information, such as the access token and refresh token, effectively logging the user out.
```
Comment (ncjones) [inline imsv-docs-astro/src/content/docs/guides/authentication.mdoc:170]:
```suggestion
Client Cleanup: The client application may discard the stored session information such as the access token and refresh token.
```

File: imsv-docs-docusaurus/openapi/endpoints/login/logout.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      content:
+        application/json:
+          schema:
+            $ref: "./models/logout-response.yaml"
+    "401":
+      description: Unauthorized or missing authentication credentials.
+      content:
+        application/json:
+          schema:
+            $ref: "../../models/api-error-401.yaml"
```
Comment (Null000) [inline imsv-docs-docusaurus/openapi/endpoints/login/logout.yaml:21]: Technically every endpoint can return a 401.
If we can add it in one place for all of them then great. If we need to add it to each endpoint, then I would do it as a separate PR and leave it out in this one.

File: imsv-docs-docusaurus/openapi/endpoints/login/models/logout-response.yaml [hunk 6]
```diff
@@ -0,0 +1,2 @@
+type: object
+description: No content returned. Successful logout is indicated by a 200 OK status code.
```
Comment (Null000) [inline imsv-docs-docusaurus/openapi/endpoints/login/models/logout-response.yaml:2]: An empty object will be returned.

File: imsv-docs-docusaurus/openapi/endpoints/login/logout.yaml (hunk 5 above)
Comment (magsharon) [reply imsv-docs-docusaurus/openapi/endpoints/login/logout.yaml]: Removed

File: imsv-docs-docusaurus/openapi/endpoints/login/models/logout-response.yaml (hunk 6 above)
Comment (magsharon) [reply imsv-docs-docusaurus/openapi/endpoints/login/models/logout-response.yaml]: changed

**Review by NicBathgate — APPROVED** [review]
Looks good!
